met-api/src/met_api/schemas/comment.py

```diff
@@ -35,19 +35,37 @@ def get_comment_reviewed_by(self, obj):
         return obj.submission.reviewed_by
 
     def get_comment_label(self, obj):
-        """Get the associated label of the comment."""
-        components = list(obj.survey.form_json.get('components', []))
-        if len(components) == 0:
-            return None
-        component_label = [
-            component.get(
-                'label',
-                None) for component in components if component.get(
-                'key',
-                None) == obj.component_id]
-        if len(component_label) == 0:
-            return None
-        return component_label[0]
+        """Get the associated label of the comment for a single page survey."""
+        form_type = obj.survey.form_json.get('display')
+        if form_type == 'form':
+            components = list(obj.survey.form_json.get('components', []))
+            if len(components) == 0:
+                return None
+            component_label = [
+                component.get(
+                    'label',
+                    None) for component in components if component.get(
+                    'key',
+                    None) == obj.component_id]
+            if len(component_label) == 0:
+                return None
+            return component_label[0]
+
+        """Get the associated label of the comment for a multi page survey."""
+        if form_type == 'wizard':
+            pages = list(obj.survey.form_json.get('components', []))
+            for page in pages:
+                components = list(page['components'])
+                if len(components) == 0:
```

Comment:
Can you create a separated method for the logic here between lines 59 and 68? it looks like its almost the same code from the block above